pkg/embed/helpers.go

```diff
@@ -53,14 +52,8 @@ func EmbedReleaseDataInBinary(binPath string, releasePath string, outputPath str
 	return nil
 }
 
-// TODO NOW: write unit tests for this
 // extractReleaseDataFromBinary extracts the release data from the binary.
-func extractReleaseDataFromBinary() ([]byte, error) {
-	exe, err := os.Executable()
-	if err != nil {
-		return nil, fmt.Errorf("unable to get executable path: %w", err)
-	}
-
+func extractReleaseDataFromBinary(exe string) ([]byte, error) {
```

Comment:
hmm, i wonder if i should make this function base64 decode as well since the embed function base64 encodes

Reply:
that would make sense to me

Reply:
done